### PR TITLE
fixed drag api

### PR DESCRIPTION
**Pull request by priyanshurohilla**
Again opened for #935

### CODE REVIEW

**Comment by astrojuanlu** [comment]
@wumpus If you approve, I will merge this one :)

File: tests/tests_twobody/test_perturbations.py [hunk 1]
```diff
@@ -191,9 +191,8 @@ def test_atmospheric_drag():
 
     # parameters of a body
     C_D = 2.2  # dimentionless (any value would do)
-    A = ((np.pi / 4.0) * (u.m ** 2)).to(u.km ** 2).value  # km^2
-    m = 100  # kg
-    B = C_D * A / m
+    A_over_m = ((np.pi / 4.0) * (u.m ** 2)).to(u.km ** 2).value / 100  # km^2/kg
```
Comment (wumpus) [inline tests/tests_twobody/test_perturbations.py:194]: These units are missing u.kg -- won't change the test result, but a future reader might be confused.

File: tests/tests_twobody/test_perturbations.py [hunk 2]
```diff
@@ -234,8 +232,7 @@ def test_atmospheric_demise():
 
     # parameters of a body
     C_D = 2.2  # dimentionless (any value would do)
-    A = ((np.pi / 4.0) * (u.m ** 2)).to(u.km ** 2).value  # km^2
-    m = 100  # kg
+    A_over_m = ((np.pi / 4.0) * (u.m ** 2)).to(u.km ** 2).value / 100  # km^2/kg
```
Comment (wumpus) [inline tests/tests_twobody/test_perturbations.py:235]: units, again

File: tests/tests_twobody/test_perturbations.py [hunk 3]
```diff
@@ -277,8 +273,7 @@ def test_atmospheric_demise_coesa76():
...
```
Comment (wumpus) [inline tests/tests_twobody/test_perturbations.py:276]: units

**Comment by priyanshurohilla** [comment]
For time being it would be better to merge this PR. I will make these necessary changes in the next PR which i will be doing for #938 as these issues are similar and will be affecting the same files

**Comment by astrojuanlu** [comment]
> For time being it would be better to merge this PR. I will make these necessary changes in the next PR which i will be doing for #938 as these issues are similar and will be affecting the same files.

What about doing all the changes here instead?

**Comment by priyanshurohilla** [comment]
> > For time being it would be better to merge this PR. I will make these necessary changes in the next PR which i will be doing for #938 as these issues are similar and will be affecting the same files.
> 
> What about doing all the changes here instead?

@astrojuanlu  That would be great 🚀 but I would like to keep them separate as i have already fixed that issue taking `poliastro/master` as the base repository :)

**Comment by astrojuanlu** [comment]
@priyanshurohilla I don't quite understand if the problem is in your git setup and I am not entirely comfortable merging a PR where there are some reasonable improvements to be made on it, but on the other hand I don't think discussing it any longer is worth the keystrokes :) The PR is OK, so I merged it. If you have any problems with git, please let us know in the appropriate channels so we can guide you without affecting the workflow of the project. Thanks!

**Comment by priyanshurohilla** [comment]
> @priyanshurohilla I don't quite understand if the problem is in your git setup and I am not entirely comfortable merging a PR where there are some reasonable improvements to be made on it, but on the other hand I don't think discussing it any longer is worth the keystrokes :) The PR is OK, so I merged it. If you have any problems with git, please let us know in the appropriate channels so we can guide you without affecting the workflow of the project. Thanks!

Actually I was unable to make you understand😅. The problem is not with the git setup, but with the way i approached it to solve the issue. I also myself don't want the PR to be merged if there are some improvements to be made. Anyways, whatever i did, i have learnt a lot now from my mistakes, so from next time onwards this thing won't happen😊. 
 
And yeah the next PR is coming soon:) 🚀.

**Pull request merged**